addons/website_membership/__openerp__.py

```diff
@@ -1,20 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Associations: Members',
     'summary': 'Online Directory of Members',
     'category': 'Website',
     'summary': 'Publish Associations, Groups and Memberships',
-    'version': '1.0',
     'description': """
 Website for browsing Associations, Groups and Memberships
 =========================================================
 """,
-    'depends': ['website_partner', 'website_google_map', 'association', 'website_sale'],
+    'depends': ['website_google_map', 'association', 'website_sale'],
```

Comment:
@hmo-odoo  Actually "website_partner" is already depends in "website_google_map" that's why i have removed it.